machinery-helper/.gitignore

```diff
@@ -0,0 +1,2 @@
+machinery-helper
```

Comment:
We decided to use machinery-helper instead so "go build" creates the correct file name without parameters.